tests/simulate/test_lammps.py

```diff
@@ -110,34 +140,59 @@ def create_file(self):
 
         return file_
 
-    def test_initialize(self):
+    def test_initialize_from_lammps_file(self):
         """Test running initialization simulation operations."""
         # InitializeFromFile
         ens, pot = self.ens_pot()
-        file_ = self.create_file()
+        file_ = self.create_lammps_file()
         op = relentless.simulate.InitializeFromFile(filename=file_)
         lmp = relentless.simulate.LAMMPS(
             op,
-            dimension=self.dim,
             types={"1": 1, "2": 2},
             executable=self.executable,
         )
         lmp.run(potentials=pot, directory=self.directory)
 
         # InitializeRandomly
         op = relentless.simulate.InitializeRandomly(seed=1, N=ens.N, V=ens.V, T=ens.T)
+        lmp = relentless.simulate.LAMMPS(op, executable=self.executable)
+        lmp.run(potentials=pot, directory=self.directory)
+
+    def test_initialize_from_gsd_file(self):
+        """Test running initialization simulation operations."""
+        if self.dim == 3:
+            ens = relentless.model.Ensemble(
+                T=2.0, V=relentless.model.Cube(L=10.0), N={"A": 2, "B": 3}
+            )
+        elif self.dim == 2:
+            ens = relentless.model.Ensemble(
+                T=2.0, V=relentless.model.Square(L=10.0), N={"A": 2, "B": 3}
+            )
+        else:
+            raise ValueError("LAMMPS supports 2d and 3d simulations")
+        ens.P = 2.5
+        pot = LinPot(ens.types, params=("m",))
+        for pair in pot.coeff:
+            pot.coeff[pair].update({"m": -2.0, "rmax": 1.0})
+        pots = relentless.simulate.Potentials()
+        pots.pair = relentless.simulate.PairPotentialTabulator(
+            pot, start=1e-6, stop=2.0, num=10, neighbor_buffer=0.1
+        )
```

Comment:
I'm not quite following why this is here?

Reply:
I had that there because of the type mismatch (HOOMD being A&B and LAMMPS being 1&2). Using with the `ens_pot()` was throwing an error

Reply:
Oh, I see! At minimum, we could add an argument to `ens_pot` that specifies whether the types are A & B or 1 & 2. Alternatively, we could revise the LAMMPS tests to use A & B (probably it is an artifact from older code that didn't support non-numeric types in LAMMPS), but that might be a little more tedious.